contracts/PushComm/PushCommEthStorageV2.sol

```diff
@@ -0,0 +1,39 @@
+pragma solidity ^0.8.20;
+
+import { CommTypes } from "../libraries/DataTypes.sol";
+
+contract PushCommEthStorageV2 {
+    /**
+     * MAPPINGS *
+     */
+    mapping(address => CommTypes.User) public users;
+    mapping(address => uint256) public nonces;
+    mapping(uint256 => address) public mapAddressUsers;
+    mapping(address => mapping(address => string)) public userToChannelNotifs;
+    mapping(address => mapping(address => bool)) public delegatedNotificationSenders;
+
+    /**
+     * STATE VARIABLES *
+     */
+    address public governance;
+    address public pushChannelAdmin;
+    uint256 public chainID;
+    uint256 public usersCount;
+    bool public isMigrationComplete;
+    address public EPNSCoreAddress;
+    string public chainName;
+    string public constant name = "EPNS COMM V1";
+    bytes32 public constant NAME_HASH = keccak256(bytes(name));
+    bytes32 public constant DOMAIN_TYPEHASH =
+        keccak256("EIP712Domain(string name,uint256 chainId,address verifyingContract)");
+    bytes32 public constant SUBSCRIBE_TYPEHASH =
+        keccak256("Subscribe(address channel,address subscriber,uint256 nonce,uint256 expiry)");
+    bytes32 public constant UNSUBSCRIBE_TYPEHASH =
+        keccak256("Unsubscribe(address channel,address subscriber,uint256 nonce,uint256 expiry)");
+    bytes32 public constant SEND_NOTIFICATION_TYPEHASH =
+        keccak256("SendNotification(address channel,address recipient,bytes identity,uint256 nonce,uint256 expiry)");
+    // New State Variables
+    address public PUSH_TOKEN_ADDRESS;
+
+    mapping(address => CommTypes.ChatDetails) public userChatData;
```

Comment:
I checked. although we don't use this variable in v2, we cannot move it or delete it since its already taking up space in the Comm v2 currently . @Zartaj0 

Although, ( something to check and verify ):

* I don't think this mapping is storing any state in Comm V2. ( double-check)
* If that's the case, we can simply replace this with another mapping and ( most likely walletToPGP) and proceed.
* Last option: we leave this state as is and add new variables below it.

Once confirmed , lemme know and i can make required changes accordingly. @Zartaj0

Reply:
ChatDetails and userChatData has now been removed